plugins/modules/fetch.py

```diff
@@ -28,6 +28,10 @@
       - This is an optional argument which provide the directory path in which the fetched
         yang modules will be saved. The name of the file is same as that of the yang module
         name prefixed with `.yang` extension.
+  continue_on_failure:
+    description:
+      - This is an optional arguement that allows schema fetch to continue if one the
+        desired models fails download
```

Comment:
Since it is a boolean type it's better to add the type of the option.
```
  continue_on_failure:
    description:
      - This is an optional argument that allows schema fetch to continue if one the
        desired models fail download
    type: bool
    default: false
```

Reply:
@ganeshrn makes sense.  Fixed in ec64ed1